README.md

```diff
@@ -16,12 +16,52 @@ A flexible JSON object linter with out of the box support for OpenAPI Specificat
 
 ## Installation
 
-```shell
+### API
+
+```bash
 npm install @stoplight/spectral
 ```
 
+### CLI
+
+```bash
+npm install -g @stoplight/spectral
+```
+
 Supports Node v8.3+ and modern browsers.
 
+### Executable binaries
+
+For users without Node and/or NPM/Yarn, we provide standalone packages for all major platforms:
+
+- x64 Windows
+- x64 MacOS
+- x64 Linux
+
+Note, the binaries are **not** auto-updatable, therefore you will need to download a new version on your own.
+
+Use of binaries is fairly easy.
```

Comment:
How do they download them? 

Can we get rid of "fairly easy" because if anyone ever struggles they could feel patronized.

Maybe move the `binaries are **not** auto-updatable` to the bottom and make it italic, its more of a footnote than leading content.

Reply:
@philsturgeon Could you enable releases tab?

Reply:
:man_facepalming: we have it.